components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -77,19 +169,63 @@ const TotalMoneyManagedSection = ({ host, isLoading }) => {
       <Container mt={18} mb={12}>
         <ProportionalAreaChart areas={chartAreas} />
       </Container>
-      <P minHeight="18px" fontSize="12px" fontWeight="400" lineHeight="18px" pt={12} pb={16}>
-        <FormattedMessage
-          id="Host.Metrics.TotalMoneyManages.description"
-          defaultMessage="Total amount held in your bank account for the Host and its Collectives."
-        />
-      </P>
+      <Flex flexWrap="wrap" justifyContent="space-between">
+        <Container px={2}>
+          <P fontSize="12px" fontWeight="400" mt="16px">
+            <FormattedMessage
+              id="Host.Metrics.TotalMoneyManages.description"
+              defaultMessage="Total amount held in your bank account for the Host and its Collectives."
+            />
+          </P>
+        </Container>
+        <Container px={2} textAlign="right">
+          <StyledLinkButton color="#46347F" asLink onClick={() => setShowMoneyManagedChart(!showMoneyManagedChart)}>
+            <P fontSize="12px" fontWeight="400" mt="16px">
+              <FormattedMessage defaultMessage="See historical" />
```

Comment:
From latest feedback about wording

```suggestion
              <FormattedMessage defaultMessage="See historic" />
```